components/Search/forms/SearchCasesForm.jsx

```diff
@@ -120,6 +122,26 @@ const SearchCasesForm = ({ onFormSubmit, defaultValues }) => {
       </div>
 
       <Button label="Search" type="submit" disabled={!isDirty} />
+      <span
+        className="govuk-body"
```

Comment:
Let's make it `govuk-link` so it more consistent with all the other links.
```suggestion
        className="govuk-link"
```